libs/remix-ui/solidity-compiler/src/lib/compiler-container.tsx

```diff
@@ -8,6 +8,8 @@ import { compilerReducer, compilerInitialState } from './reducers/compiler'
 import { resetEditorMode, listenToEvents } from './actions/compiler'
 import { OverlayTrigger, Tooltip } from 'react-bootstrap' // eslint-disable-line
 import { getValidLanguage } from '@remix-project/remix-solidity'
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import * as json_config from '../../../../../apps/remix-ide/contracts/solidity_compiler_config.json'
```

Comment:
could you put this file in the `solidity-compiler` react plugin or in the remix-solidity module?